eng/common-tests/matrix-generator/samples/matrix.json

```diff
@@ -4,7 +4,7 @@
   },
   "matrix": {
     "Agent": {
-      "ubuntu": { "OSVmImage": "ubuntu-18.04", "Pool": "Azure Pipelines" },
+      "ubuntu": { "OSVmImage": "ubuntu-20.04", "Pool": "Azure Pipelines" },
```

Comment:
```suggestion
      "ubuntu": { "OSVmImage": "ubuntu-22.04", "Pool": "Azure Pipelines" },
```

Reply:
Unless this was intentional to only go to 20.04! In which case I shutup.

Reply:
Actually I'm seeing this a lot. Going to reach out offline. Ignore this comment chain!